gostream/tools/tools.go

```diff
@@ -1,12 +0,0 @@
-//go:build tools
```

Comment:
We are removing this file because RDK has its own linting

Reply:
yes.